hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/ErrorsAndExceptionsEnricher.java

```diff
@@ -158,5 +167,18 @@ public void enrichTrace(StructuredTrace trace) {
               EnrichedSpanConstants.getValue(ErrorMetrics.ERROR_METRICS_TOTAL_SPANS_WITH_ERRORS),
               MetricValueCreator.create(errorCount));
     }
+
+    Map<String, AttributeValue> enrichedAttributes = new HashMap<>();
+
+    ApiTraceGraph apiTraceGraph = new ApiTraceGraph(trace);
+    for (ApiNode<Event> apiNode : apiTraceGraph.getApiNodeList()) {
+      Optional<Event> entryEvent = apiNode.getEntryApiBoundaryEvent();
+      int apiTraceErrorCount =
+          (int) apiNode.getEvents().stream().filter(this::findIfEventHasError).count();
+      if (entryEvent.isPresent()) {
+        enrichedAttributes.put(
```

Comment:
I don't think the intention was to achieve this. I think you wanted to enrich the enriched attributes of the `entryEvent`. The `enrichedAttributes` currently is just a new map instantiated